client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -179,11 +185,11 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
                 >
                   <InputLabel sx={{ fontSize: '12px' }}>
                     {packSizeController.selected?.value === -1
-                      ? t('label.units-in-pack-size-of', {
+                      ? `${t('label.unit-plural', {
                           unit,
                           count: quantity,
-                        })
-                      : t('label.packs-of', { count: quantity })}
+                        })} ${t('label.in-packs-of')}`
+                      : t('label.in-packs-of', { count: quantity })}
```

Comment:
this block (lines 188-192) should be:

```
                      ? `${t('label.unit-plural', {
                          unit,
                          count: issueQuantity,
                        })} ${t('label.in-packs-of')}`
                      : t('label.in-packs-of')}
```

the quantity should be `issueQuantity` and `label.in-packs-of` has no params now